pkg/upgrade/main.go

```diff
@@ -37,3 +47,7 @@ func versions(versions map[string]upgradeFunction) ([]*semver.Version, error) {
 	sort.Sort(semver.Collection(versionLists))
 	return versionLists, nil
 }
+
+func noop(ctx context.Context, client client.Client, jaeger v1.Jaeger) (v1.Jaeger, error) {
```

Comment:
As only used for testing, can't this go into the `main_test.go` file?

Reply:
Yes